contributors/guide/code-review.md

```diff
@@ -0,0 +1,275 @@
+care. It should document why it cannot be further reduced or changed. For an
+example of this, see the _"[KEEP THE SPACE SHUTTLE FLYING]"_ portion of the
+persistent volume controller as an example.
+
+
+### Ensure you properly test your changes
+
+Code submitted without tests will almost always be rejected at first pass. If an
+area you're updating did not previously have them, adding them in your PR will
+be appreciated and helps improve the overall health of the project.
+
+If something you change has the potential to impact the performance or scalability
+of the project, include a benchmark. These are essential to demonstrate the 
+efficacy of your change and will help in preventing potential regressions.
+
+
+### Commits messages should be thorough and detailed
+
+PR comments are not represented in the commit history. Important details
+regarding your changes should be kept in your commit message.
+
+The commit message subject line should be no more than 72 characters and
+summarize the changes being made.
+
+Use the body of the commit message to describe **WHY** the change is being made.
+Describing why something has changed is more important than how. You are
+providing context to both your reviewer and the next person that has to touch
+your code.
+
+For more guidance on commit messages, see our [commit message guidelines].
+
+
+### Pull Request comments should summarize the PR
+
+The GitHub pull request should contain a summary of the changes being made as
+well as links to all the relevant information related to the PR itself. These
+are things such as issues it closes, KEPs, and any other relevant documentation.
+
+Accurately filling out the issue template and applying the necessary labels
+will greatly speed up the initial triage of your PR.
+
+If there is a portion of your PR that should be explained or linked that is not
+committed as a comment, you can review your own PR, applying additional context
+or links.
+
+
+### Minor changes (typos / link fixes)
+
+While these sort of fixes are appreciated, try and fix multiple instances at the
+same time if possible, instead of issuing subsequent individual PRs for each fix.
+
+---
+
+## Tips for code reviewers
+
+### Managing time
+
+#### Block time off to review
+
+Often it can be hard to find dedicated, uninterrupted time to review PRs. If you
+can, allocate some time and block it off on your calendar to help stay on top of
+the incoming queue.
+
+#### Taking a break or stepping away
+
+If you are taking a break, going on vacation, or stepping away for a bit -- you
+can set your [GitHub status] to busy; this will signal [Blunderbuss] to not
+automatically assign you to reviews (It does not block manual assignment).
+
+If for an extended period of time, or if you need to focus on other areas,
+consider setting yourself as an `[emeritus_approver]` in some of the areas your
+an approver.
+
+
+### Loop in others when Domain Specific Knowledge is needed
```

Comment:
nit: the capitalization of the domain specific knowledge phrase feels strange emphasis